modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/AuditGraphApp.js

```diff
@@ -12,20 +12,42 @@
  * details.
  */
 
+import ClayAlert from '@clayui/alert';
 import React from 'react';
 
 import AuditBarChart from './components/AuditBarChart';
 import EmptyAuditBarChart from './components/EmptyAuditBarChart';
 
 export default function ({context, props}) {
-	const {languageDirection} = context;
+	const {languageDirection, namespace} = context;
 	const {learnHowURL, vocabularies} = props;
 
+	const hasBarsCategoryFilters = () => {
+		return new URLSearchParams(window.location.href).get(
+			'resetBarsCategoryFiltersURL'
+		);
+	};
```

Comment:
I don't think you really need to create a function here. You could just set it to a variable like below

```js
const hasBarsCategoryFilters = new URLSearchParams(window.location.href).get('resetBarsCategoryFiltersURL');
```

Also, I don't think [URLSearchParams is supported in ie11](https://caniuse.com/urlsearchparams). You might be able to get away with similar logic you have below `uri.includes('resetBarsCategoryFiltersURL')`. Although maybe IE isn't a concern for this.

Regardless of IE, its probably best to use the same means of checking this value. Either `URLSearchParams` or `href.includes()`

Reply:
I think we are going with `URLSearchParams` and support Edge and other browsers, not IE. I'll change the code to use `URLSearchParams` instead of `.includes`. Thanks for the feedback!